app/policies/supported_permission_policy.rb

```diff
@@ -0,0 +1,16 @@
+class SupportedPermissionPolicy < BasePolicy
+  class Scope < ::BasePolicy::Scope
+    def resolve
+      if current_user.superadmin?
+        scope.all
+      elsif current_user.admin?
+        scope.all
+      elsif current_user.organisation_admin?
+        app_ids = Pundit.policy_scope(current_user, :manage_supported_permissions_for_application).pluck(:id)
```

Comment:
I think `Access` is wrong that word is used elsewhere to mean the applications that the user has been granted `signin` permission to, and that's not what this list is for, it's the list of applications the user is allowed to grant permissions to other users for.  One candidate therefor might be `ApplicationsUserCanGrantPermissionsToOtherUsersForPolicy`; it's cumbersome for sure but it is more obvious than what I had before.  On the other hand this logic did come from a method called `visible_applications` so we could use that `VisibleApplicationsPolicy`, but it's not intention revealing at all.  I'll have another think.